src/pz390.java

```diff
@@ -13875,14 +13936,28 @@ private int get_long_log_sub_cc() {
 			}
 		}
 	}
-
-	private byte[] get_log_bytes(byte[] data_byte, int data_offset, int data_len) {
+	private byte[] get_unsigned_bytes(byte[] data_byte, int data_offset, int data_len) {  // #300
+		/*
+		 * return byte array with leading 0 byte followed by data bytes. This
+		 * array format is used to initialize BigInteger with logical unsigned
+		 * value.  
+		 * 2022-02-21 dsh issue 300 support unsigned and signed 2s compliment negative numbers with leading x'ff' vs x'00'
```

Comment:
I think you added this comment when you were first addressing the issue about signed numbers. It should be deleted.